src/main.cpp

```diff
@@ -19,13 +23,11 @@
 #include "definitions.h"
 
 // local libs
-#include "../interfaces/abstract_task.h"
```

Comment:
Registering it as a component (above) would allow to include the header as usual i.e. `#include <abstract_task.h>`...

Reply:
Please register it --> Issue #85